### PR TITLE
feat(main-nav): Main Nav refactoring, refactor the logo

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

In the condensed view of the Left Menu it shows the new logo. The logo is no longer a link to the home page, but just a container with an image

### Why is it needed?

In the refactoring of the Main Nav we want a new home icon to have the task of connecting to the home page, while the logo must only show the image of the logo.
The Home icon will be implemented in another PR.

### How to test it?
Disclaimer: the new logo is used only in the condensed view at the moment because I need to implement the components used in the Logo and in the new Home icon before implementing the new MainNav wrapper

1. Open a Strapi App
2. in the left Main Nav click on the reduce button, to show the condensed view of the Main Nav
3. you can now see the new version of the logo (only the width is not correct because we need to change the width of the MainNav wrapper)
4. if you try to click on the logo nothing happens

You can find all the info and link to the figma file in the Jira ticket

### Related issue(s)/PR(s)

CS-762

### CODE REVIEW

File: packages/core/admin/admin/src/components/LeftMenu.tsx [hunk 1]
```diff
@@ -105,23 +107,45 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }: LeftMenuProps) =
...
+          })}
+          title={menuTitle}
+          icon={
+            <img
+              src={menu.custom?.url || menu.default}
+              alt={formatMessage({
+                id: 'app.components.LeftMenu.logo.alt',
+                defaultMessage: 'Application logo',
+              })}
+            />
```
[the remark this answers is not in the document]
Comment (simotae14) [reply packages/core/admin/admin/src/components/LeftMenu.tsx]: Thanks for the suggestion, you are right we can add all the props data directly into the new NavBrand, let me know if you like the refactored version

File: packages/core/admin/admin/src/components/MainNav/tests/NavBrand.test.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    logos: {
+      menu: {
+        default: 'default',
+        custom: {
+          url: 'custom',
+        },
+      },
+    },
+  }),
+}));
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/components/MainNav/tests/NavBrand.test.tsx:17]: I think we have a `msw` endpoint for this, if you await one of the assertions it should work? 👀

**Review by joshuaellis — APPROVED** [review]
Minor comment, but looks good, nice and clean 🧽

File: packages/core/admin/admin/src/components/MainNav/NavBrand.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  }
+`;
+
+export const NavBrand = () => {
+  const { formatMessage } = useIntl();
+  const {
+    logos: { menu },
+  } = useConfiguration('LeftMenu');
+  return (
+    <Box paddingLeft={3} paddingRight={3} paddingTop={3} paddingBottom={3}>
```
Comment (Marc-Roig) [inline packages/core/admin/admin/src/components/MainNav/NavBrand.tsx:24]: I will just ask probably dumb questions, so feel free to correct me as much as you want

Does the box compo has a "padding" attribute to do something like <Box padding={3}> ?

**Review by madhurisandbhor — COMMENTED** [review]
Screenshot 2024-04-18 at 15 35 12
I see some misalignment 🤔

**Comment by simotae14** [comment]
yes @madhurisandbhor you are right but for the moment the MainNav width in condensed mode is wrong, I am gonna work on it in another ticket, now you need just to give a look to the logo and not the wrapper

**Review by madhurisandbhor — APPROVED** [review]

**Comment by joshuaellis** [comment]
> yes @madhurisandbhor you are right but for the moment the MainNav width in condensed mode is wrong, I am gonna work on it in another ticket, now you need just to give a look to the logo and not the wrapper

when you do the wrapper, can you redo the CSS of the icon, because if you do a whole wrapper you wanna add the padding to the container and then use gap for it's children imo, but for now this is fine :)

**Pull request merged**